openedx/features/assessment/helpers.py

```diff
@@ -38,7 +38,7 @@ def find_and_autoscore_submissions(enrollments, submission_uuids):
         None
     """
     days_to_wait = configuration_helpers.get_value('DAYS_TO_WAIT_AUTO_ASSESSMENT', DAYS_TO_WAIT_AUTO_ASSESSMENT)
-    delta_date = datetime.now(utc).date() - timedelta(days=days_to_wait)
+    delta_date = datetime.now(utc) - timedelta(days=days_to_wait)
```

Comment:
now this variable `delta_date ` contains datetime instead of date, right? can we rename it so that it gives proper info?